.github/workflows/push.yml

```
@@ -68,7 +66,29 @@ jobs:
           push: true
           tags: ghcr.io/${{ github.repository }}:${{ needs.branch_meta.outputs.sha }}
           labels: ${{ steps.docker_meta_img.outputs.labels }}
-
+      - name: Docker meta Service Name (file storage)
+        id: docker_meta_img_file_storage
+        uses: crazy-max/ghaction-docker-meta@v4
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=ref,event=branch,enable=false,priority=600
+            type=sha,enable=true,priority=600,prefix=
```

Add the prefix `file-storage-` also here
```suggestion
            type=ref,event=branch,enable=false,priority=600,prefix=file-storage-
            type=sha,enable=true,priority=600,prefix=file-storage-
```